internal/security/credentials_test.go

```diff
@@ -126,8 +133,8 @@ func TestMain(m *testing.M) {
 		getSecretsTestObj{
 			testName:          "Valid key (key1 not cached)",
 			path:              "db_secrets",
-			keys:              []string{"key1"},
-			expectedSecrets:   map[string]string{"key1": "value1"},
+			keys:              []string{"key1", "key2"},
```

Comment:
did you modify this test because of the error when running the package tests?

Reply:
Yes, let's investigate this more on Monday

Reply:
@eno-intel I re-tested this and it works the original way, so I reverted my change.